### PR TITLE
Add Spanish translation

### PR DESCRIPTION
**Pull request by xoan**

### CODE REVIEW

File: po/es.po [hunk 1]
```diff
@@ -0,0 +1,175 @@
+# Spanish translation for Gcolor3
+# Copyright (C) 2013-2016 Jente Hidskes <hjdskes@gmail.com>
```
Comment (Hjdskes) [inline po/es.po]: Shouldn't this be you, in 2018? ;)

**Comment by Hjdskes** [comment]
Thanks for the PR! I'll let you fix that copyright statement, and then I'll merge it :)

File: po/es.po (hunk 1 above)
Comment (xoan) [reply po/es.po]: I guess not. This copyright statement is related to you, the PACKAGE COPYRIGHT HOLDER.

Below the next line, my name is added as a translator.

But probably, the year for your (C) must be changed, right?

**PS**. Glibc is a good example: https://sourceware.org/git/?p=glibc.git;a=blob;f=po/es.po;hb=HEAD

Comment (Hjdskes) [reply po/es.po]: Alright, I'll merge this then and fix those lines in all .po files later.

**Pull request merged**